### ISSUE TITLE
Mysterious Obelisk message doesn't count as dialogue

### ISSUE THREAD

**Issue by Mcrtin** [Wynntils/Wynntils#1951]
![image](https://github.com/Wynntils/Artemis/assets/66840021/405921ef-56ac-4344-8071-ae9f8612e131)

**Comment by magicus** [comment]
Well, should it..? We do not and cannot capture all Wynn messages that are printed to the chat. There is in principle no way for us to know that this is part of the quest.

Unless you see the message on a new "chat page" on vanilla, this is intended behavior.

**Comment by Mcrtin** [comment]
it resends the messages from before as you can see in the screenshot after the "as you touch..." message. Idk what you would call that but I don't like the resend messages and maybe we can redirect the "as you..." to Notafications.

**Comment by magicus** [comment]
Ok, we'll have to investigate. Are there any requirements for interacting with this obelisk? (I've missed it, is it part of the time valley mini bosses thing?)

**Comment by Mcrtin** [comment]
idk i just click on it with my main (100% quests before 2.0)

**Comment by Mcrtin** [comment]
"A mysterious crystal found in [Nemract](https://wynncraft.fandom.com/wiki/Nemract) and [Tempo Town](https://wynncraft.fandom.com/wiki/Tempo_Town). It requires the completion of [Tempo Town Trouble](https://wynncraft.fandom.com/wiki/Tempo_Town_Trouble)."https://wynncraft.fandom.com/wiki/Fast_Travel#Mysterious_Obelisk

**Comment by MenacingPerson** [comment]
This was added in 2.0.3

**Comment by kristofbolyai** [comment]
Yes, but supporting is edge case is rather hard, for no benefit..

**Comment by magicus** [comment]
I am a bit skeptical if this can/should be fixed, but I'll go and have a look at the obelisk dialogue myself first before closing this.

**Comment by Mcrtin** [comment]
This can only be matched by hard coding the string

**Comment by magicus** [comment]
If that is the only solution, then we will not fix it. However, you stated before:

> it resends the messages from before as you can see in the screenshot after the "as you touch..." message.

and that matches a general pattern how Wynn treats NPC messages. This is behavior that we can match on (and, in general, we ought to already do).

**Comment by Mcrtin** [comment]
The messages are resent but only after a few seconds so during that time we have no idea that the message is dialogue if we don't match the exact string and therefore it can't be displayed as one.

**Comment by magicus** [comment]
I checked this out myself now. The bug report is correct, but your speculation to the cause is way off.

Wynncraft sends "chat screens" as for NPC dialogues here, but for some reason, we fail to detect when the chat screen is done. It has nothing to do with the actual message being sent.

**Comment by Mcrtin** [comment]
Thx for the confirmation.
I didn't know that we have a way of detecting something like that, but I'm glad we do. :)
